docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud.md

```diff
@@ -0,0 +1,491 @@
+           for example `org123a456b789`. Get your `ORGANIZATION_ID`
+           [from the Aiven Console or CLI](/docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud#prerequisites).
+         - `CUSTOM_CLOUD_ID` with the identifier of your custom cloud, which you can
+           extract from the output of the [avn byoc list](/docs/tools/cli/byoc#avn-byoc-list)
+           command, for example `018b6442-c602-42bc-b63d-438026133f60`.
+
+      - [avn byoc template terraform get-vars](/docs/tools/cli/byoc#avn-byoc-template-terraform-get-vars)
+
+            ```bash
+            avn byoc template terraform get-vars              \
+              --organization-id "ORGANIZATION_ID"     \
+              --byoc-id "CUSTOM_CLOUD_ID" >| "tf_dir/tf_file.vars"
+            ```
+
+         Replace the following:
+
+         - `ORGANIZATION_ID` with the ID of your Aiven organization to
+           connect with your own cloud account to create the custom cloud,
+           for example `org123a456b789`. Get your `ORGANIZATION_ID`
+           [from the Aiven Console or CLI](/docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud#prerequisites).
+         - `CUSTOM_CLOUD_ID` with the identifier of your custom cloud, which you can
+           extract from the output of the [avn byoc list](/docs/tools/cli/byoc#avn-byoc-list)
+           command, for example `018b6442-c602-42bc-b63d-438026133f60`.
+
+   1. Optionally, modify the template as needed.
+
+        :::note
+        To connect to a custom-cloud service from different security groups
+        (other than the one dedicated for the custom cloud) or from IP
+        address ranges, add specific ingress rules before you apply a
+        Terraform infrastructure template in your Google Cloud account in the process
+        of creating a custom cloud resources.
+
+        Before adding ingress rules, see the examples provided in the
+        Terraform template you generated and downloaded from the [Aiven
+        Console](https://console.aiven.io/).
+        :::
+
+   1. Use Terraform to deploy the infrastructure template with the provided variables in
+      your Google Cloud account. This will generate a privilege-bearing service account (SA).
+
+       :::important
+       When running `terraform plan` and `terraform apply`, add `-var-file=FILE_NAME.vars`
+       as an option.
+       :::
+
+   1. Find `privilege_bearing_service_account_id` in the output script after running
+      the template.
+
+1. Provision resources by running [avn byoc provision](/docs/tools/cli/byoc#avn-byoc-provision)
+   and passing the generated `google-privilege-bearing-service-account-id` as an option.
+
+        ```bash
+        avn byoc provision                            \
+          --organization-id "ORGANIZATION_ID" \
+          --byoc-id "CUSTOM_CLOUD_ID"         \
+          --google-privilege-bearing-service-account-id "GENERATED_SERVICE_ACCOUNT_ID"
```

Comment:
It would be more pretty to align all (everywhere in the docs) the end of line `\` backslashes.